### PR TITLE
add integration python examples horovod

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by Le-Zheng** [comment]
Jenkins: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Python-integration-test/1/console

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    echo "Clearing up environment. Uninstalling analytics-zoo"
+    pip uninstall -y analytics-zoo
+    pip uninstall -y bigdl
+    pip uninstall -y pyspark
+}
+
+echo "#1 Start ray horovod pytorch example tests"
+start=$(date "+%s")
+# run example
+python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/orca/learn/horovod/pytorch_estimator.py --cluster_mode yarn-client
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:12]: add --num_nodes 2 ? Default is only one node

**Comment by hkvision** [comment]
We can merge it first and let owners add them separately.

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    clear_up
+    echo "ray horovod pytorch failed"
+    exit $exit_status
+fi
+
+now=$(date "+%s")
+time1=$((now-start))
+
+echo "#1 pytorch estimator example time used:$time1 seconds"
+
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:25]: Also clear up at the very end?

**Review by hkvision — APPROVED** [review]

File: pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    clear_up
+    echo "ray horovod pytorch failed"
+    exit $exit_status
+fi
+
+now=$(date "+%s")
+time1=$((now-start))
+
+# This should be done at the very end after all tests finish.
+clear_up
```
Comment (yangw1234) [inline pyzoo/zoo/examples/run-example-test-ray-horovod-pytorch-integration.sh:25]: I think it is a little bit strange to only `uninstall` here without `installing`. Maybe we can move the uninstalling logic to jenkins job, or move the installing logic in jenkins job to here.

**Pull request merged**